programs/management/commands/add_il_programs.py

```diff
@@ -0,0 +1,81 @@
+from django.core.management.base import BaseCommand
+from programs.models import Program, FederalPoveryLimit, LegalStatus
+from screener.models import WhiteLabel
+
+
+class Command(BaseCommand):
+    help = "Add Illinois programs to database"
```

Comment:
experimenting with what it would look like to import the programs for white label into the DB. haven't gotten this to work yet. please ignore this file.